### PR TITLE
#168835355 When sign in or create an account only the token is returned to the user

### PR DESCRIPTION
**Pull request by patricktunez123**
#### What does this PR do?
Have only token returned to the user when signing up or creates a new account
#### Description of Task to be completed?
Have the following endpoints returning only a token to the user
POST /api/v1/auth/signup
POST /api/v1/auth/signin
#### How should this be manually tested?
After cloning the repo, cd into it and run node server
Using postman test /api/v1/auth/signup and /api/v1/auth/signin only the token should be returned
#### What are the relevant pivotal tracker stories?
#168835355

### CODE REVIEW

File: server/helpers/signin.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const login = {
     const loginvalidation = {
```
Comment (ekumamait) [inline server/helpers/signin.js]: I think the best thing to do in this case would be to check if the password/email matches not validating it again! Why are you validating them again?

**Review by ekumamait — CHANGES_REQUESTED** [review]
Please work on this and give me a response

File: server/helpers/signin.js (hunk 1 above)
Comment (patricktunez123) [reply server/helpers/signin.js]: Oh thank you so much for such feedback, that is true! i should not be validating a user for signing him/her in. Thanks!

**Review by ekumamait — APPROVED** [review]
Go ahead and merge

**Pull request merged**